packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,204 +24,262 @@ _ref:
+            The config object has the following properties:
 
-            The global object can also be modified using the [`SetGlobal`](/SetGlobal) action.
+            - `basePath: string`: Set the base path to serve the Lowdefy application from. This will route all pages under `https://example.com/<base-path>/<page-id>` instead of the default `https://example.com/<page-id>`. The basePath value must start with "/".
+            - `homePageId: string`: The pageId of the page that should be loaded when a user loads the app without a pageId in the url route. This is the page that is loaded when you navigate to `yourdomain.com`.
 
-            # Menus
+            ## Auth
 
-            Menu objects are an object with links to pages. The are used by blocks like `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+            [TODO]: <> (# TODO: Add auth things here)
 
-            Blocks like like `PageSiderMenu` use the menu with `id: default` by default. This means that if you create a menu object with `id: default`, this will be used unless another menu is configured.
+            ## Global
 
-            More than one menu can be configured in an app. As an example, this can be used when two logically different sections in the app need different menus.
+            Any data that you wish to use in your app can be stored in the __global__ object, and accessed using the [`_global`](/_global) operator. This is a good place to store data or configuration that is used throughout the app, for example the url of a logo or configuration of a page, since then these are only written once, and can be updated easily.
 
-            If OpenID Connect authentication and role based authentication is configured, pages that the user is restricted from seeing are filtered from the menu object.
+            The global object can also be modified using the [`SetGlobal`](/SetGlobal) action.
 
-            The schema for a menu object is:
+            ## Connections
 
-            - `id: string`: __Required__ - A identifier for the menu. If it is `default`, it will be used as default by pages.
-            - `links: object[]`: An array of `MenuLink` or `MenuGroup` objects that form the links in the menu. `MenuGroups` can be two levels deep.
+            In a Lowdefy app you can integrate with other services like API's or databases using `connections` and `requests`. Connections configure the connection settings to the service, and often contain parameters like connection strings, urls and secrets such as passwords or API keys. Requests are used to interact with the connection, for example inserting a data record, executing a query or calling a API end-point.
```

Comment:
... [`connections` and `requests`](/connections-and-requests) ...
just check the page url.

Reply:
Change has been made in #1314.